src/main/webapp/app/shared/components/code-button/code-button.component.ts

```diff
@@ -347,4 +357,69 @@ export class CodeButtonComponent implements OnInit, OnChanges {
             this.user.vcsAccessToken = this.activeParticipation.vcsAccessToken;
         }
     }
+
+    initTheia(profileInfo: ProfileInfo) {
```

Comment:
I think you could make this method private

Reply:
Done, good point